services/core-web/src/components/mine/Incidents/MineIncidentTable.js

```diff
@@ -218,10 +241,11 @@ export class MineIncidentTable extends Component {
       },
       {
         title: "EMPR Action",
+        key: "followup_action",
         dataIndex: "followup_action",
-        className: hideColumn(this.props.isDashboardView),
+        className: hideColumn(true),
         render: (action, record) => (
-          <div title="EMPR Action" className={hideColumn(this.props.isDashboardView)}>
+          <div title="EMPR Action" className={hideColumn(true)}>
```

Comment:
Aaron wants to hide the EMPR Action column in both views for now. Now, the Status column is displayed for the mine-specific view.